docs/source/examples/sequence_tagging.rst

```diff
@@ -292,15 +295,11 @@ Since our sequences are of variable lengths and we want to be the most efficient
             """
                 Defines the computation performed at every call.
             """
-            total_length = padded_sequences_vectors.shape[1]
-
-            pack_padded_sequences_vectors = pack_padded_sequence(padded_sequences_vectors, lengths.cpu(), batch_first=True)
-
             lstm_out, self.hidden_state = self.lstm_network(pack_padded_sequences_vectors)
-            lstm_out, _ = pad_packed_sequence(lstm_out, batch_first=True, total_length=total_length)
+            lstm_out, _ = pad_packed_sequence(lstm_out, batch_first=True)
 
-            tag_space = self.fully_connected_network(lstm_out)
-            return tag_space.transpose(-1, 1) # we need to transpose since it's a sequence
+            tag_space = self.fully_connected_network(lstm_out.data)
```

Comment:
Why `.data`?

Reply:
Since the lstm_out is a packed_sequence again and we only want to use the data in it (the output of the LSTM).

Reply:
But just before, you do `pad_packed_sequence`?

Reply:
Yes. This `self.lstm_network(pack_padded_sequences_vectors)` return a packed_sequence. It needs to be repacked to match the proper dimension. It's not possible to just remove the following: `pad_packed_sequence(lstm_out, batch_first=True)`. Since the dimension won't fit the batch size. 

Packed sentences are but in a different order than the original sequence. The sequence is moved in a column format instead of row format. Thus the first row of the first tensor is all the first token of all sentences and so on. Thus, the dimension is different, and we need to reconstruct the proper order for the fully_connected network.

Reply:
According to the doc, [`pad_packed_sequence`](https://pytorch.org/docs/stable/generated/torch.nn.utils.rnn.pad_packed_sequence.html?highlight=pad_packed_sequence#torch.nn.utils.rnn.pad_packed_sequence) returns tensors.

Reply:
True. 
Don't who why is it there. Maybe a previous test.

Reply:
678f745